lib/ZumoOta/src/WebServerCustom.cpp

```diff
@@ -69,9 +70,49 @@ WebServerCustom::~WebServerCustom()
 {
 }
 
+String WebServerCustom::listFiles(bool ishtml)
+{
+  String returnText = "";
+  LOG_DEBUG("Listing files stored on LittleFS");
+  File root = LittleFS.open("/");
+  File foundfile = root.openNextFile();
+  if (ishtml) {
+    returnText += "<table><tr><th align='left'>Name</th><th align='left'>Size</th></tr>";
+  }
+  while (foundfile) {
+    if (ishtml) {
+      returnText += "<tr align='left'><td>" + String(foundfile.name()) + "</td><td>" + humanReadableSize(foundfile.size()) + "</td></tr>";
+    } else {
+      returnText += "File: " + String(foundfile.name()) + "\n";
+    }
+    foundfile = root.openNextFile();
+  }
+  if (ishtml) {
+    returnText += "</table>";
+  }
+  root.close();
+  foundfile.close();
+  return returnText;
+}
+
+String WebServerCustom:: humanReadableSize(const size_t bytes)
+{
+  if (bytes < 1024) return String(bytes) + " B";
+  else if (bytes < (1024 * 1024)) return String(bytes / 1024.0) + " KB";
+  else if (bytes < (1024 * 1024 * 1024)) return String(bytes / 1024.0 / 1024.0) + " MB";
+  else return String(bytes / 1024.0 / 1024.0 / 1024.0) + " GB";
+}
+
 void WebServerCustom::init()
 {
-    
+   server.on("/filelist", HTTP_GET, [this](AsyncWebServerRequest *request)
+    {
+        String fileList = listFiles(true);
+        request->send(200,"text/html", fileList);
+    });
+
+    // ... Weitere Routen und Konfigurationen ...

```

Comment:
stick to english please

Reply:
It will be taken care of by the next commit.